seatunnel-connectors-v2/connector-starrocks/src/main/java/org/apache/seatunnel/connectors/seatunnel/starrocks/util/SchemaUtils.java

```diff
@@ -0,0 +1,321 @@
+        try (Statement statement = connection.createStatement()) {
+            log.info("Executing change column SQL: " + changeColumnSQL);
+            statement.execute(changeColumnSQL);
+        }
+    }
+
+    public static void applySchemaChange(
+            Connection connection, TablePath tablePath, AlterTableModifyColumnEvent event)
+            throws SQLException {
+        String tableIdentifierWithQuoted = tablePath.getFullName();
+        Column modifyColumn = event.getColumn();
+        String afterColumn = event.getAfterColumn();
+        String modifyColumnSQL =
+                buildAlterTableSql(
+                        AlterType.MODIFY.name(),
+                        modifyColumn,
+                        tableIdentifierWithQuoted,
+                        StringUtils.EMPTY,
+                        afterColumn);
+
+        try (Statement statement = connection.createStatement()) {
+            log.info("Executing modify column SQL: " + modifyColumnSQL);
+            statement.execute(modifyColumnSQL);
+        }
+    }
+
+    public static void applySchemaChange(
+            Connection connection, TablePath tablePath, AlterTableAddColumnEvent event)
+            throws SQLException {
+        String tableIdentifierWithQuoted = tablePath.getFullName();
+        Column addColumn = event.getColumn();
+        String afterColumn = event.getAfterColumn();
+        String addColumnSQL =
+                buildAlterTableSql(
+                        AlterType.ADD.name(),
+                        addColumn,
+                        tableIdentifierWithQuoted,
+                        StringUtils.EMPTY,
+                        afterColumn);
+        try (Statement statement = connection.createStatement()) {
+            log.info("Executing add column SQL: " + addColumnSQL);
+            statement.execute(addColumnSQL);
+        }
+    }
+
+    public static void applySchemaChange(
+            Connection connection, TablePath tablePath, AlterTableDropColumnEvent event)
+            throws SQLException {
+        String tableIdentifierWithQuoted = tablePath.getFullName();
+        String dropColumn = event.getColumn();
+        String dropColumnSQL =
+                buildAlterTableSql(
+                        AlterType.DROP.name(), null, tableIdentifierWithQuoted, dropColumn, null);
+        try (Statement statement = connection.createStatement()) {
+            log.info("Executing drop column SQL: " + dropColumnSQL);
+            statement.execute(dropColumnSQL);
+        }
+    }
+
+    /**
+     * build alter table sql
+     *
+     * @param alterOperation alter operation of ddl
+     * @param newColumn new column after ddl
+     * @param tableName table name of sink table
+     * @param oldColumnName old column name before ddl
+     * @param afterColumn column before the new column
+     * @return alter table sql for sink table after schema change
+     */
+    public static String buildAlterTableSql(
```

Comment:
It doesn't need to be so complicated, please simplify the code.

reference
#8134

Reply:
ok,I optimizing the code,thanks.